standalone/standalone.tpl

```diff
@@ -97,7 +97,7 @@ MODEL_DTYPE = "bfloat16"
 MAX_WORKERS = "auto"
 MERGE_SYSTEM_USER_MESSAGE = False
 FEW_SHOTS = 5
-BATCH_SIZE = 8
+BATCH_SIZE = "auto"
```

Comment:
You need to add " " around the variable here as well https://github.com/redhat-et/ilab-on-ocp/blob/fcc26a7c6b61ee9d0f7d767369e6345dd9cb40ac/pipeline.py#L501

Reply:
done!